src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -182,7 +181,7 @@ public final void testPathsDirectoryWithNestedFile() throws IOException {
         antTask.setProject(new Project());
 
         final FileResource fileResource = new FileResource(
-            antTask.getProject(), getPath(FLAWLESS_INPUT_DIR));
+            antTask.getProject(), getPath(FLAWLESS_INPUT));
```

Comment:
@subkrish If you can modify test to fail with your incorrect changes, that would be great.